testing/trino-testing/src/main/java/io/trino/testing/AbstractTestDistributedQueries.java

```diff
@@ -597,6 +597,20 @@ public void testInsertUnicode()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testInsertHighestUnicode()
```

Comment:
Idea for a better method name? We are specifically testing U+10FFF in this method.

Reply:
`testInsertHighestCharacter` could be slightly better.